Shoko.Server/Providers/TMDB/TmdbMetadataService.cs

```diff
@@ -2031,6 +2031,50 @@ private void PurgeCompany(int companyId, bool removeImageFiles = true)
 
     #region People
 
+    public async Task<(int Found, int Updated, int Skipped, int Removed)> RepairMissingPersons(bool removeErrors = false)
+    {
+        var missingIds = new HashSet<int>();
+        var (updateCount, skippedCount, removedCount) = (0, 0, 0);
+        
+        var people = _tmdbPeople.GetAll().Select(person => person.TmdbPersonID).ToList();
+        
+        _tmdbEpisodeCast.GetAll().AsParallel()
```

Comment:
Drop the parallel in these, and do a `for each(…¹)` loop instead of `.ForAll`. we don't need to use _all_ available threads for this. you can convert the `.Where` into an if statement inside the `for each`. be it an `if(…¹) continue;` statement or an `if(…¹) …²` statement.